### ISSUE TITLE
Fix up styling on Homework Excuse Generator lesson plan.

### ISSUE THREAD

**Issue by flukeout** [mozilla/thimble.mozilla.org#2208]
Via email, user pointed out that the text that is circled below is hard to read.

![image](https://cloud.githubusercontent.com/assets/25212/26462690/9d6cdf56-4136-11e7-8cae-99a84727e6c9.png)

**Comment by wintop6211** [comment]
I can fix this! Assign it to me thanks!

**Comment by humphd** [comment]
@wintop6211 I'm not sure if you'll be able to fix this, since the code isn't in github, I think you'd have to have access to the original project.  Maybe @flukeout can chime in.

**Comment by gideonthomas** [comment]
Yeah we manage that account internally so I don't think this is a good issue for contributors.
